src/components/FormCharDetails.js

```diff
@@ -32,82 +33,107 @@ const CREATE_DETAILS = gql`
     ) {
       id
       name
+      species
```

Comment:
Adding the career, specialization, etc. here should allow you to access their values in the mutation response!

Reply:
Gotchu, thanks!